packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -354,16 +353,21 @@ export default function DataviewsPatterns() {
 		return filterSortAndPaginate( patterns, viewWithoutFilters, fields );
 	}, [ patterns, view, fields, type ] );
 
-	const templatePartActions = usePostActions( TEMPLATE_PART_POST_TYPE );
-	const patternActions = usePostActions( PATTERN_TYPES.user );
-	const editAction = useEditPostAction();
+	const templatePartActions = usePostActions( {
+		postType: TEMPLATE_PART_POST_TYPE,
+		context: 'view',
+	} );
+	const patternActions = usePostActions( {
+		postType: PATTERN_TYPES.user,
+		context: 'view',
+	} );
 
 	const actions = useMemo( () => {
 		if ( type === TEMPLATE_PART_POST_TYPE ) {
-			return [ editAction, ...templatePartActions ].filter( Boolean );
+			return templatePartActions.filter( Boolean );
```

Comment:
Is filtering still doing anything useful? I'd be surprised if `usePostActions` were returning empty values.